src/wallet/wallet2.cpp

```diff
@@ -10167,9 +10167,9 @@ std::string wallet2::make_uri(const std::string &address, const std::string &pay
 //----------------------------------------------------------------------------------------------------
 bool wallet2::parse_uri(const std::string &uri, std::string &address, std::string &payment_id, uint64_t &amount, std::string &tx_description, std::string &recipient_name, std::vector<std::string> &unknown_parameters, std::string &error)
 {
-  if (uri.substr(0, 7) != "monero:")
+  if (uri.substr(0, 7) != "wownero:")
```

Comment:
You need to use `uri.substr(0, 8)` because the lengths of strings "monero" and "wownero" are different.

Reply:
Thanks stoffu, added suggested changes to release-v0.2.1.0 branch.